### DISCUSSION TITLE
Line breaks </br> no longer working, but <br> and double space are fine.

### DISCUSSION THREAD

**Discussion by rmscode** [mkdocs/mkdocs#3849] [Q&A]
For the past year and a half since I started using mkdocs, I always used `</br>` for line breaks. I noticed this morning after building my site . . . that they aren't being respected. I could just replace all of them with `<br>` or double spaces I guess, but that's kind of a pain. Does mkdocs handle line breaks on its own or is that part of a python extension that mkdocs uses? I don't know where to begin investigating.

_This . . ._

```
Line 1</br>
Line 2<br>
Line 3 (Double space)     
Line 4
```

_Produces this . . ._

Line 1 Line 2
Line 3
Line 4

You can see that the line break missing from the final html:

![image](https://github.com/user-attachments/assets/4ea3596e-5614-46ce-9b49-b7226ddbdf97)

**Comment by pawamoy** [comment]
I cannot reproduce the issue you describe. `</br>` correctly create line breaks in the final HTML pages. If `<br>` works on your side, then a quick regex-replace should be easy.

We can try to help further if you're able to provide a minimal reproducible example :slightly_smiling_face: (a tiny project that demonstrates the issue).

**Reply by rmscode** [reply]
I appreciate the reply and offer to help. If I can come up with something, I will present it here.

**Reply by pawamoy** [reply]
Try to isolate the issue: it could come from enabled plugins or Markdown extensions :smile:

**Reply by rmscode** [reply]
I have narrowed down the issue. I recently installed the [mkdocs-audio](https://github.com/jfcmontmorency/mkdocs-audio) plugin. I will report the issue to the maintainer. If you would like to reproduce this yourself, all you need is a fresh instance of mkdocs and that plugin.
